cloud: add TiDB Cloud CLI docs (#12111)
pingcap/docs: tidb-cloud/ticloud-config-set.md

---
title: ticloud config set
Summary: The reference of `ticloud config set`.
---

# ticloud config set

> You can use `ticloud config set` to configure the properties for the active [user profile](/tidb-cloud/cli-reference.md#user-profile) as follows:

Reviewer: ```suggestion
Configure the properties for the active [user profile](/tidb-cloud/cli-reference.md#user-profile):
```

Reviewer: Change reason: To keep consistent style with the short description of other commands.

```shell
ticloud config set <property-name> <value> [flags]
```

> The property names that can be configured include `public-key`, `private-key`, and `api-url`.

Reviewer: ```suggestion
The properties that can be configured include `public-key`, `private-key`, and `api-url`.
```

|-------------|--------------------------------------------------------------------|----------|
| public-key  | The public key of the TiDB Cloud API                               | Yes      |
| private-key | The private key of the TiDB Cloud API                              | Yes      |
| api-url     | The base API URL of TiDB Cloud (`https://api.tidbcloud.com` by default) | No       |

> **Notes:**
>
> If you want to configure properties for a specific user profile, you can add the `-P` flag and specify the target user profile name in the command.

## Examples

Set the value of the public-key for the active profile:

```shell
ticloud config set public-key <public-key>
```

Set the value of the public-key for a specific profile `test`:

```shell
ticloud config set public-key <public-key> -P test
```

Set the API host:

```shell
ticloud config set api-url https://api.tidbcloud.com
```

> **Note:**
>
> The TiDB Cloud API URL is `https://api.tidbcloud.com` by default. Usually, you do not need to set up it.

## Flags

| Flag       | Description              |
|------------|--------------------------|

> | -h, --help | Gets the help information for this command |

Reviewer: ```suggestion
| -h, --help | Help information for this command |
```
